blockchain/fundamentals/anonimity/how-to-de-anonymize-bitcoin.md

```diff
@@ -0,0 +1,59 @@
+# How to De-Anonymize Bitcoin
+author: mihaiberq
+
+levels:
+
+  - basic
+
+  - medium
+
+type: normal
+
+category: must-know
+
+tags:
+
+  - introduction
+
+  - deep
+
+links:
+  - '[Bitcoin transactions are not that anonymous.](https://www.technologyreview.com/s/608716/bitcoin-transactions-arent-as-anonymous-as-everyone-hoped/)'
+
+---
+## Content
+
+In the previous insight, we mentioned a specific way of linking transactions to an online identity: joint payments. Most of the times, however, a single redeemed UTXO is enough to track someone if the transacted value is specific enough.
+
+A bitcoin transaction can represent up to 8 digits after the decimal points (`0.000000001 bitcoin` is called a `satoshi`), so the distribution space is large. If someone is to know the exact value of your transaction and a timeframe in which you had intiated the transaction, there's a chance they could track your address.
+
+And depending on where they found out about the exact value of the transaction, they can even know your real identity. A barista that shows you the coffeeshop's public address can also link your identity to the transaction. By donating bitcoins and sharing the donation confirmation to incentivize others to do the same, you are also revealing your blockchain identity.
+
+Why should you care though? Well, chances are you won't have that many bitcoins anyway. If you made a good investment and have a large sum of money in your bank account, there's a slim chance of anyone knowing it unless you told someone. But your Bitcoin address is public and will always be public.
+
+If you decide to buy more bitcoins because you belive it is the coin of the future, what stops others from trying to steal your private key? Attention to details and precaution go a long way. To make the whole tracking process harder, *mixing services*, or simply mixers, appeared. 
```

Comment:
Add some comments here about 
- mixers not protecting you fully
- how mixers actually work